### DISCUSSION TITLE
Backblaze with 2FA

### DISCUSSION THREAD

**Discussion by devanteweary** [JonathanTreffler/backblaze-personal-wine-container#93] [Q&A] [answered]
Hey!!

Just wondering how you use this with 2FA activated on your Backblaze account.

I don't have it activated yet because I'm scared to lose access to any data, so just wondering if this is compatible with 2FA.

Thank you!

**Comment by JonathanTreffler** [comment, marked as answer]
I have not tested it recently, but it should be compatible.

**Reply by devanteweary** [reply]
I tested it yesterday and seems to work.
